providers/summary/clearlydefined.js

```diff
@@ -280,22 +291,9 @@ class ClearlyDescribedSummarizer {
     if (manifest.dist && manifest.dist.url) {
       setIfValue(result, 'described.urls.download', manifest.dist.url)
     }
-    // We could have singular licenses such as 'MIT' or licenses in an array ['MIT', 'BSD']
-    // Process licenses depending on whether they are strings or array of strings
-    let expression = null  // license expression (as input to SPDX.normalize)
-    if (typeof manifest.license === 'string') {
-      expression = manifest.license
-    } else if (Array.isArray(manifest.license)) {
-      expression = manifest.license.join(' OR ')
-    } else if (typeof manifest.licenses === 'string') {
-      expression = manifest.licenses // handle legacy NPM 'licenses' key
-    } else if (Array.isArray(manifest.licenses)) {
-      expression = manifest.licenses.join(' OR ')
-    }
-    if (expression) {
-      const licenses = SPDX.normalize(expression)
-      setIfValue(result, 'licensed.declared', licenses)
-    }
+    const expression = this.parseLicenseExpression(manifest, 'OR')
+    if (!expression) return
+    setIfValue(result, 'licensed.declared', SPDX.normalize(expression))
```

Comment:
Looking at the composer data, it seems like there is no licenses element in the manifest. Has it been mentioned anywhere in the documentation? If not, please revert the changes in this function to the original code.

Reply:
So I ran a script over all dev & production composer definitions and it looks like they do not have licenses field. There is only the license field. Have you also seen any occurrences of license array with objects and not strings?  For the sake of reducing the complexity, can we go back to the original code for the composer type:
```
if (Array.isArray(manifest.license)) {
  const licenses = SPDX.normalize((manifest.license || []).join(' OR '))
  setIfValue(result, 'licensed.declared', licenses)
}
```

Reply:
I just added a fix to not bother with manifest.licenses for composer... Please see if this is acceptable.